src/js/paste.js

```diff
@@ -145,7 +145,7 @@ var PasteHandler;
                     }
                 }
             } else {
-                this.pasteHTML(text);
+                this.pasteHTML(text, this.base.options);
```

Comment:
What was the intended result of passing this into `pasteHTML()`?  As far as I can tell, nothing in `pasteHTML` actually uses any of the values expose from `this.base.options`